Unittests are failing with urllib3>1.26

Unittests are failing with error:
```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <botocore.awsrequest.AWSHTTPConnection object at 0x7f9c856d7278>

header = 'Host', values = (b'169.254.169.254',)

    def putheader(self, header, *values):

        """"""

>       if SKIP_HEADER not in values:

E       BytesWarning: Comparison between bytes and string

.tox/3.7-linux-unit/lib/python3.7/site-packages/urllib3/connection.py:218: BytesWarning
```
It seems that the issue was introduced by:
https://github.com/urllib3/urllib3/pull/2018

--- a/requirements/extras/sqs.txt
+++ b/requirements/extras/sqs.txt
@@ -1,2 +1,3 @@
 boto3>=1.4.4
 pycurl==7.43.0.2  # Latest build with wheels provided
+urllib3<1.26      # Unittests are faiing with urllib3>=1.28

